cluster/hooks.go

```diff
@@ -433,23 +433,42 @@ func RegisterDomainPostHook(input interface{}) error {
 		log.Infof("Domain '%s' already registered", domain)
 	}
 
-	_, err = InstallOrUpdateSecrets(
+	secretSources, err := InstallOrUpdateSecrets(
 		commonCluster,
-		&secretTypes.ListSecretsQuery{
+		&pkgSecret.ListSecretsQuery{
 			Type: pkgCluster.Amazon,
-			Tag:  secretTypes.TagBanzaiHidden,
+			Tag:  pkgSecret.TagBanzaiHidden,
 		},
 		route53SecretNamespace,
 	)
-
 	if err != nil {
 		log.Errorf("Failed to install route53 secret into cluster: %s", err.Error())
 		return err
 	}
 
-	log.Info("route53 secret successfully installed into cluster")
+	secretID := secret.GenerateSecretIDFromName(secretSources[0].Name)
 
-	return nil
+	route53Secret, err := secret.Store.Get(orgId, secretID)
+	if err != nil {
+		log.Errorf("Failed to install route53 get secret : %s", err.Error())
```

Comment:
Fix the log message.

Reply:
You could add a new method `GetByName(orgId, name)` to `Store` that expects a name and internally takes care of generating the secret id and invoking  `Get(orgId, secretID)`